frontend/src/metabase/account/notifications/containers/NotificationListApp.jsx

```diff
@@ -0,0 +1,26 @@
+import { connect } from "react-redux";
+import _ from "underscore";
+import Alerts from "metabase/entities/alerts";
+import Pulses from "metabase/entities/pulses";
+import { getUser } from "metabase/selectors/user";
+import { getUserNotifications } from "../selectors";
+import NotificationList from "../components/NotificationList";
+import { push } from "react-router-redux";
+
+const mapStateToProps = state => ({
+  user: getUser(state),
+  items: getUserNotifications(state),
+});
+
+const mapDispatchToProps = {
+  onShowHelp: () => push("/account/notifications/help"),
+};
+
+export default _.compose(
+  Alerts.loadList(),
```

Comment:
Once BE support it, we should add a `user_id` or something like this here to filter out all unneeded notifications.